workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/StarredDropdown.tsx

```diff
@@ -0,0 +1,167 @@
+
+/**
+ * @public
+ * Props for each starred entitify item
+ */
+interface SectionComponentProps {
+  entityRef: string | CompoundEntityRef | Entity;
+  toggleStarredEntity: (
+    entityOrRef: Entity | CompoundEntityRef | string,
+  ) => void;
+  handleClose: () => void;
+}
+
+const StarredItem: React.FC<SectionComponentProps> = ({
+  entityRef,
+  toggleStarredEntity,
+  handleClose,
+}) => {
+  const { Icon, primaryTitle, secondaryTitle } =
+    useEntityPresentation(entityRef);
+  const { name, kind, namespace } = parseEntityRef(entityRef as string);
+  const theme = useTheme();
+
+  return (
+    <MenuItem
+      component={Link}
+      to={`/catalog/${namespace || 'default'}/${kind}/${name}`}
+      onClick={handleClose}
+      disableRipple
+      disableTouchRipple
+    >
+      {Icon && (
+        <ListItemIcon sx={{ minWidth: 36 }}>
+          <Icon />
+        </ListItemIcon>
+      )}
+      <ListItemText
+        primary={
+          <Typography sx={{ color: theme.palette.text.primary }}>
+            {primaryTitle || secondaryTitle}
+          </Typography>
+        }
+        secondary={kind.toLocaleUpperCase()}
+        // inset={!Icon}
+        sx={{ ml: 1, mr: 1 }}
+      />
+      <Tooltip title="Remove from list">
+        <IconButton
+          onClick={e => {
+            e.preventDefault();
+            e.stopPropagation();
+            toggleStarredEntity(entityRef);
+          }}
+        >
+          <Star color="warning" />
+        </IconButton>
+      </Tooltip>
+    </MenuItem>
+  );
+};
+
+const NoStarredItems = () => {
+  return (
+    <InfoCard>
+      <Box
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="center"
+        textAlign="center"
+        py={4}
+        px={2} // Added padding to control width
+        maxWidth={300} // Set max width to constrain text expansion
+        mx="auto"
+      >
+        <AutoAwesomeIcon sx={{ fontSize: 64 }} color="disabled" />
+        <Typography variant="h6" color="textPrimary" mt={2}>
+          No starred items yet
+        </Typography>
+        <Typography variant="body2" color="textSecondary" mt={1} maxWidth={250}>
+          Click the star icon next to an entity's name to save it here for quick
+          access.
+        </Typography>
+      </Box>
+    </InfoCard>
+  );
+};
+
+export const StarredDropdown = () => {
+  const { anchorEl, handleOpen, handleClose } = useDropdownManager();
+  const { starredEntities, toggleStarredEntity } = useStarredEntities();
+
+  const entitiesArray = Array.from(starredEntities);
+
+  return (
+    <HeaderDropdownComponent
+      buttonContent={<StarBorderIcon />}
+      onOpen={handleOpen}
+      onClose={handleClose}
+      anchorEl={anchorEl}
+      tooltip="Your starred items"
+      isIconButton
+    >
+      {entitiesArray.length > 0 ? (
+        <>
+          <ListItemText
+            primary="Your starred items"
+            sx={{ pl: 2, fontWeight: 'bold' }}
```

Comment:
Minor style changes.
```suggestion
            sx={{ pl: 2, mt: 1, fontWeight: 'bold', color: 'text.secondary' }}
```

Reply:
Done.